railties/lib/rails/generators/rails/app/templates/config/application.rb

```diff
@@ -34,6 +34,10 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+<%- elsif !depends_on_system_test? -%>
+
+    # Don't generate system test files.
+    config.generators.system_tests = nil
```

Comment:
We can.  
But, like the integration tool, I think that it is a `string` so that the user can customize it.
https://github.com/rails/rails/blob/master/railties/lib/rails/test_unit/railtie.rb#L13..L14